vertical-pod-autoscaler/pkg/recommender/logic/recommender.go

```diff
@@ -34,6 +34,7 @@ var (
 	targetMemoryPercentile     = flag.Float64("target-memory-percentile", 0.9, "Memory usage percentile that will be used as a base for memory target recommendation. Doesn't affect memory lower bound nor memory upper bound.")
 	lowerBoundMemoryPercentile = flag.Float64("recommendation-lower-bound-memory-percentile", 0.5, `Memory usage percentile that will be used for the lower bound on memory recommendation.`)
 	upperBoundMemoryPercentile = flag.Float64("recommendation-upper-bound-memory-percentile", 0.95, `Memory usage percentile that will be used for the upper bound on memory recommendation.`)
+	humanizeMemory             = flag.Bool("humanize-memory", false, "Convert memory values in recommendations to the highest appropriate SI unit without decimal places for better readability.")
```

Comment:
I guess this help string needs a tweak, now that it's 2 decimal places

Reply:
Yeap, thanks!

Reply:
Yeap, thanks!